client/libs/core-state/src/lib/index.ts

```diff
@@ -3,15 +3,18 @@ import { ActionReducerMap } from '@ngrx/store';
 import * as fromSermons from './sermons/sermons.reducer';
 import * as fromSpeakers from './speakers/speakers.reducer';
 import * as fromTags from './tags/tags.reducer';
+import * as fromMedia from './media/media.reducer';
 
 export interface AppState {
   sermons: fromSermons.State;
```

Comment:
I know its not part of your PR, but can you update the name for `State` from `sermons` to be like all of the others.

So, `sermons: fromSermons.State` to `sermons: fromSermons.SermonsState`. Then, of course, update the actual state interface name in the reducer.